app/views/users/index.html.erb

```diff
@@ -1 +1,19 @@
-<h1>Here is a list of Users</h1>
\ No newline at end of file
+<h1>Welcome to Users page</h1>
+
+<div class="container">
+    <ul>
+        <% @users.each do |user| %>
+            <%= link_to user do %>
+                <li class="card">
+                    <div class="img">
+                        <img src=<%= user.photo %> alt="user photo" width="100%" height="100%">
+                    </div>
+                    <div class="content">
+                        <h2><%= user.name %></h2>
+                        <p>Number of post: <%= user.posts_counter %></p>
+                    </div>
+                </li>
+            <% end %>
+        <% end %>
```

Comment:
- [ ] [optional] Great work on the user's page, it would be nice to use ` app/views/users/_user.html.erb` as best practice instead of using `@users.each`, the end files will be like

`    
     # app/views/users/_user.html.erb
 
     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
`